.travis.yml

```diff
@@ -18,6 +21,9 @@ install:
  - go get github.com/golang/lint/golint
  - go get github.com/fzipp/gocyclo
 
+before_script:
+ - openssl req -x509 -newkey rsa:4096 -keyout server.key -out server.crt -days 365 -nodes -subj /CN=localhost
```

Comment:
Maybe add a comment along the lines of:
```
# Generate a self-signed X.509 certificate for TLS.
```

 I'm not entirely sure that openssl is self-explanatory enough for the casual reader.